### DISCUSSION TITLE
twrp刷包时报错，代码255

### DISCUSSION THREAD

**Discussion by smilencecc** [LSPosed/LSPosed#1938] [Q&A] [answered]
twrp模式刷入1.8.3zip压缩包时报错，错误代码25
![1](https://user-images.githubusercontent.com/77428601/168257001-9fb64adf-07c9-4ebb-8b70-8a472f807f72.jpg)
5

**Comment by yujincheng08** [comment, marked as answer]
从来都没有支持过 twrp 刷入

**Reply by smilencecc** [reply]
> 从来都没有支持过 twrp 刷入

这个zip包不就是twrp模式刷入的么，我以前都是这么刷进去的

**Reply by maidmeow4** [reply]
不是的，你需要在Magisk里安装这个zip。

**Reply by smilencecc** [reply]
> 不是的，你需要在Magisk里安装这个zip。

好的  感谢  已经刷进去了

**Comment by Seasickil-CN** [comment]
twrp先把magisk刷进去，把你要刷的lsposed包从magisk卡刷进去，没有lsposed管理器的话，在所刷的包里找管理器的apk文件，解压出来装好就行了，至于zymagisk或者riru你自己选择

**Reply by smilencecc** [reply]
好的  我试试
